### ISSUE TITLE
[BUG] Can join a public/group chat when you're already a member 

### ISSUE THREAD

**Issue by Nikateen** [neelkamath/omni-chat-web#30]
No check to see if you're already a member. 

However this doesn't seem to cause any other issues with regards to duplicate members in groups.

**Comment by neelkamath** [comment]
The code is actually fine (i.e., this was intentional) but as you said, I think it's a little confusing because it makes it look as if the user isn't already a participant. I wasn't too sure what to do instead. Should we have the "Join" button be disabled with text like "Already a participant" instead?

**Comment by Nikateen** [comment]
Yes, I think that is ideal, but it's also alright to just change the message you receive when it redirects you to the chat page to something like "Already a participant"
